plugins/scanners/dos/src/main/kotlin/DosScanner.kt

```diff
@@ -228,7 +228,10 @@ class DosScanner internal constructor(
                 }
 
                 "failed" -> {
-                    issues += createAndLogIssue(name, "Scan failed in DOS API")
+                    issues += createAndLogIssue(
```

Comment:
Out of scope: Would it make sense to also log the `jobId` when 1. starting, 2. completing or 3. failing?

Reply:
Yeah true, that's actually quite helpful too, I added a commit here.